shared/feature.go

```diff
@@ -24,14 +24,36 @@ type Feature struct {
 }
 
 func NewFeature(feature *models.Feature) Feature{
+	var f Feature
+	f.Meta = NewMeta(feature.Meta)
+	f.Snippet = feature.Snippet
+	f.TestSnippet = feature.TestSnippet
+	return f
+}
+
+func NewFeature_str(name string, desc string, auth string, updated time.Time, dependencies []string, snippet string, testSnippet string) Feature{
```

Comment:
Not sure that `_str` is a reasonable name here since time is passed using `time.Time` field